pageTests/admin/archive-a-ward-success.test.js

```diff
@@ -0,0 +1,31 @@
+import { getServerSideProps } from "../../pages/admin/archive-a-ward-success";
+
+jest.mock("../../src/usecases/adminIsAuthenticated", () => () => (token) =>
```

Comment:
If we aren't moving this now then bring the TODO along too.

Reply:
I'm not sure, I didn't want to refactor all this in my tests. I'll bring the todo.